src/styles/calendar.less

```diff
@@ -144,3 +144,20 @@ div.ant-picker-calendar-date-value {
 .ant-picker-ok button {
   position: relative !important;
 }
+
+.calendar-event-modal .ant-modal-footer {
+  text-align: center;
+  :hover {
+    color: rgba(0, 0, 0, 0.65) !important;
+  }
+  .ant-btn-primary {
+    background-color: #fcdfaa !important;
+    color: rgba(0, 0, 0, 0.65) !important;
+    margin-left: 0 !important;
+  }
+  button {
+    border: 1px solid rgba(0, 0, 0, 0.65) !important;
+    width: auto !important;
+    margin: 0 4% !important;
+  }
+}
```

Comment:
a great fix to the bug using className. Just wondereing but does it need to be named .calendar-event-button.ant-btn
do we need to put the .ant-btn?